oauth2_http/java/com/google/auth/oauth2/ComputeEngineCredentials.java

```diff
@@ -57,13 +63,16 @@
  *
  * <p>Fetches access tokens from the Google Compute Engine metadata server.
  */
-public class ComputeEngineCredentials extends GoogleCredentials {
+public class ComputeEngineCredentials extends GoogleCredentials implements ServiceAccountSigner {
 
   private static final Logger LOGGER = Logger.getLogger(ComputeEngineCredentials.class.getName());
 
   // Note: the explicit IP address is used to avoid name server resolution issues.
   static final String DEFAULT_METADATA_SERVER_URL = "http://169.254.169.254";
 
+  static final String IAM_API_ROOT_URL = "https://iam.googleapis.com/v1";
```

Comment:
nit: Do we need to expose this base url? Can we combine into `SIGN_BLOB_URL_FORMAT`?